mkdocs.yml

```diff
@@ -29,12 +30,15 @@ nav:
       - Creating: tests-creating.md
       - Running: tests-running.md
       - Getting Results: tests-getting-results.md
-      - Postman Collections: executor-postman.md
-      - Cypress Tests: executor-cypress.md
-      - Curl Commands: executor-curl.md
-      - SoapUI Tests: executor-soapui.md
   - Executors:
-      - Creating custom executor: executor-custom.md
+      - Cypress: cypress-executors.md
```

Comment:
should all the executor files have the same naming convention ?

Reply:
I agree.  I'll make that change with my next PR.